babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeJournal.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+/**
+ * A <code>SqlCreateAttributeJournal</code> is a CREATE TABLE option
+ * for the JOURNAL attribute.
+ */
+public class SqlCreateAttributeJournal extends SqlCreateAttribute {
+
+  private JournalType journalType;
+  private JournalModifier journalModifier;
+
+  /**
+   * Creates a {@code SqlCreateAttributeJournal}.
+   *
+   * @param journalType  Type of journal image to be maintained
+   * @param journalModifier  Journal image modifier
+   * @param pos  Parser position, must not be null
+   */
+  public SqlCreateAttributeJournal(JournalType journalType,
+      JournalModifier journalModifier,
+      SqlParserPos pos) {
+    super(pos);
+    this.journalType = journalType;
+    this.journalModifier = journalModifier;
+  }
+
+  @Override public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
+    switch (journalModifier) {
+    case NO:
+      writer.keyword("NO");
+      break;
+    case DUAL:
+      writer.keyword("DUAL");
+      break;
+    case LOCAL:
+      writer.keyword("LOCAL");
+      break;
+    case NOT_LOCAL:
+      writer.keyword("NOT");
+      writer.keyword("LOCAL");
+      break;
+    default:
```

Comment:
You could choose to have a case for "UNSPECIFIED" case and omit the default case, since there would not be any other case. Reference from [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.8.4-switch). Also, you might not need the "break;" in default case.

Reply:
This comment also applies to the default case in line71.

Reply:
It looks like the style guide says either is fine, so I may leave it because that's how it is in the SqlCreateTable class as well for Volatile. But I think the break could probably be removed since it doesn't do anything in this case

Reply:
Sounds good